microsoft-edge/origin-trials/index.md

```diff
@@ -13,12 +13,73 @@ You can use Origin Trials to try out experimental APIs on your own live site for
 
 To support the experimental APIs, users don't need to go to `edge://flags` on their own device to access the experimental APIs that your site uses.  Instead, you register the origin that you want to use the experimental feature on (for example, `https://www.contoso.com:443`), and you'll then be sent a token that you add to the code of your website.
 
-To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
+To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
 
 You can also provide feedback to browser engineers and the web standards community about the design of the API, your use cases, or your experience using the APIs.
 
-For more information, see the **Getting Started** and **Developer Guidelines** sections in [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).
+> [!IMPORTANT] 
```

Comment:
Convert alert to standard specific heading.

https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#alerts-note-tip-important-caution-warning - "Avoid Note, Tip, and Important alerts—readers tend to skip over them. Placing that info directly into the article text is better."

i plan to push commit